html-forms/index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <div>Email:
+      <input type="text" name="email" size="15" maxlength="30" />
+    </div>
+    <div>Why is coding cool?</div>
+    <select name="Select an option">
+      <option selected="Select">Select an option</option>
+      <option value="interesting">Because it's interesting</option>
+      <option value="fun">Because it's fun</option>
+      <option value="powerful">Because it's powerful</option>
+    </select>
+    <br />
+    <p>Favorite Language (choose one)</p>
+    <input type="radio" name="language" value="HTML" checked="checked" /> HTML
+    <input type="radio" name="language" value="CSS" checked="checked" /> CSS
+    <input type="radio" name="language" value="JavaScript" checked="checked" /> JavaScript
+
+    <p>Would you like to sign up for more information?
+      <br />
+      <br />
```

Comment:
Let's not use br tags in order to make new lines (please remove all the br tags in this file)